browser/brave_rewards/rewards_tab_helper.cc

```diff
@@ -76,13 +76,22 @@ void RewardsTabHelper::ResourceLoadComplete(
     return;
 
   // TODO(nejczdovc): do we need to get anyother type then XHR??
-  if (resource_load_info.resource_type == ResourceType::kMedia ||
-      resource_load_info.resource_type == ResourceType::kXhr ||
-      resource_load_info.resource_type == ResourceType::kImage ||
-      resource_load_info.resource_type == ResourceType::kScript) {
-    rewards_service_->OnXHRLoad(tab_id_, GURL(resource_load_info.final_url),
-                                web_contents()->GetURL(),
-                                resource_load_info.referrer);
+  switch (resource_load_info.request_destination) {
+    // Formerly ResourceType::kMedia
+    case network::mojom::RequestDestination::kAudio:
```

Comment:
this needs header and possibly deps added. Always include what you use

Reply:
also https://github.com/brave/brave-core/pull/5102/commits/ae27df94d98c898db82b24cbdd2388555d2603e9#diff-7facd19a12f2073f3d15f0fa3181c0ffR22 is that header/dep still needed?

Reply:
The file already has "#include "third_party/blink/public/mojom/loader/resource_load_info.mojom.h" and the dep in browser/BUILD.gn

The diff is not working for me, so I am not sure which header/dep